### ISSUE TITLE
Nested formatter status - missing from current version of API docs

### ISSUE THREAD

**Issue by eldjh3** [fmtlib/fmt#4111]
RE: https://github.com/fmtlib/fmt/issues/2086#issuecomment-1724536538

This comment on an earlier issue mentions the recommended way to perform formatting of nested/complex objects.
It mentions it's experimental at that point in time.

The documentation is no longer present in the API page of the docs.  It was commented out in 97117cbb51ce83905736ef4287645c99adba646c "Migrate to mkdocs".

What's the status now?  Is it's omission from the docs an oversight or is it no longer experimental but deprecated?

I was experimenting with it and noticed that some of the specifiers are applied to the constituents but some only to the complete value.  I wasn't sure if that was intentional at first.  It is in fact documented, but only in the (now commented out) section.

**Comment by eldjh3** [comment]
Incidentally, it looks like the width is actually applied to elements, contrary to what the commented out doc states.
https://godbolt.org/z/KsM5shEnY

```
Notice that fill, align and width are applied to the whole object which
is the recommended behavior while the remaining specifiers apply to
elements.
```

**Comment by vitaut** [comment]
It was removed intentionally from the docs and requires a bit more work to fix width/precision handling, see #3860.
